### DISCUSSION TITLE
Differences between JetBrains theme and the VS Code theme.

### DISCUSSION THREAD

**Discussion by gino** [arcticicestudio/nord#240] [Help & Support]
Hello,

I am wondering if this is a known issue but if I compare the JetBrains Nord theme to the Nord theme for VSCode, some colors do not look the same. Is this a known issue or is there a way (without customizing a lot) that I could achieve the same in VS Code?

## Screenshot
On the left side we have PhpStorm and on the right side we have VS Code.

![image](https://user-images.githubusercontent.com/23482089/77248432-9a8ba480-6c39-11ea-8329-6ab52230d32a.png)

The PhpStorm variant has a couple of purple/pink accents which I do not have in the VS Code one, unfortunately.

**Comment by arcticicestudio** [comment]
Hi @gino :wave:, thanks for your contribution :+1:

I guess this might be related to https://github.com/arcticicestudio/nord-jetbrains/pull/121. There were severals problems regarding the defined theme keys in the _Nord JetBrains_ port project due to changes between IDE versions. An example problem reported by a user is https://github.com/arcticicestudio/nord-jetbrains/issues/115 where functions in _WebStorm_ used the color of _WebStorm_'s default theme instead of _Nord_.
If your _PHPStorm_ setup is also affected by this problem please try to re-install the _Nord_ theme plugin and also follow the instructions posted in https://github.com/arcticicestudio/nord-jetbrains/issues/115.

**Comment by arcticicestudio** [comment]
@gino Did the instructions helped to solve the problem?

**Comment by svengreb** [comment]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.

---

@gino Based on the stale state I guess the problem has been resolved or is not relevant anymore so I‘ll close this discussion.
